pkg/scheduler/scheduler.go

```diff
@@ -287,17 +287,17 @@ func (sched *Scheduler) schedule(pod *v1.Pod) (core.ScheduleResult, error) {
 // preempt tries to create room for a pod that has failed to schedule, by preempting lower priority pods if possible.
 // If it succeeds, it adds the name of the node where preemption has happened to the pod annotations.
 // It returns the node name and an error if any.
```

Comment:
This line also needs change: It returns an error if any.

Reply:
+1. Please address.

Reply:
DONE